src/main/webapp/script.js

```diff
@@ -154,12 +156,12 @@ function passButtonPressed() {
 }
 
 function loadProfile() {
-  let id = getCurrentUserId();
-  if (id === null) {
+  let userID = getCurrentUserId();
+  if (userID === null) {
     return;
   }
-  console.log(id);
-  fetch('/user-data?id=' + id).then(response => response.json()).then((userinfo) => {
+  console.log(userID);
```

Comment:
Do you still need to log? If yes, two things:

1.  Generally you're better off including context here, not just a number (e.g., `"userID = " + userID`)
1.  Open an issue to look for and remove log statements - you don't want these in the final product